.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Release
+about: Create a tracking issue for a release
+title: 'Release 1.x.xxxx'
+labels: []
+assignees: ''
+
+---
+
+# Schedule
+
+- [ ] MM/DD/YYYY - Release branch forks from `main`
+    - At this point, changes must be cherry-picked into the release branch in
+      order for them to be included in the release.
+- [ ] MM/DD/YYYY - Release Candidate 1 (begin Ask Mode[^1] for release branch).
+    - At this point, changes must be approved by @microsoft/hlsl-release
+- [ ] MM/DD/YYYY - Final Release Candidate
+- [ ] MM/DD/YYYY - Target Release Date
+
+
+# Tasks
+
+## Before Fork
+
+This part of the release process is to 'prime the pump' - that is to make sure
+that all the various parts of the engineering system are set into place so that
+we are confident we can generate builds for the new branch
+
+- [ ] Create the release branch from `main`
+    - The release branch is kept into sync with main via regular fast-forward
+      merges.
+- [ ] Internal branches and build pipelines configured
+    - Verify that the engineering system can build:
+    - [ ] Zip files for github release
+    - [ ] NuGet package
+    - [ ] VPack
+- [ ] Final merge of `main` into the release branch
+
+## After Fork
+
+- [ ] Update README.md
+- [ ] Create draft of Release post on GitHub
+
+## Quality Sign Off
+
+- [ ] Microsoft Testing Sign-off (@pow2clk)
+- [ ] Google Testing Sign-off (@sudonatalie)
```

Comment:
Natalie is no longer involved in DXC development.

Reply:
@s-perron / @Keenuts - any thoughts on who should be the POC for signing off DXC releases these days?

Reply:
If it has to be one person, add me. If it can be multiple, add both of us. We can decide who will be responsible when the realese is about to happen.